packages/framework/esm-framework/mock.tsx

```diff
@@ -304,3 +304,6 @@ export const showModal = jest.fn();
 export const LeftNavMenu = jest.fn();
 export const setLeftNav = jest.fn();
 export const unsetLeftNav = jest.fn();
+
+/* Components */
+export {OpenmrsDatePicker} from "@openmrs/esm-styleguide";
```

Comment:
```suggestion

export { OpenmrsDatePicker } from "@openmrs/esm-styleguide";
```

The formatting looks off and I think we can do just great without the comment.